blog/incubation-announcement.md

```diff
@@ -0,0 +1,74 @@
+slug: incubation-announcement
+date: 2025-11-18
+authors: [aaguiar, jakub]
+tags: [announcement]
+image: https://openfga.dev/img/og-rich-embed.png
+hide_table_of_contents: false
+---
+# OpenFGA Accepted into CNCF Incubation 🎉
+
+OpenFGA has been accepted into the Cloud Native Computing Foundation Incubation stage. This milestone reflects years of community work and real-world adoption.
+
+- CNCF announcement: [OpenFGA becomes a CNCF Incubating Project](https://www.cncf.io/blog/2025/11/11/openfga-becomes-a-cncf-incubating-project/)
+- TOC tracker: [Issue #1949](https://github.com/cncf/toc/issues/1949)
+
+## Why This Matters
+
+Incubation is a signal that a project is healthy, vendor neutral, and ready for broad adoption. It places OpenFGA on the same maturity path as OpenTelemetry, Keycloak, Artifact Hub and Backstage.
+
+Learn more about [CNCF project stages](https://www.cncf.io/projects/).
+
+## How We Got Here
+
+- Community and learning. Two years of public community meetings, talks, and guides. Join us in [CNCF Slack](https://openfga.dev/docs/community) and find recordings on our [YouTube channel](https://www.youtube.com/@OpenFGA).
+- Adopter validation through public adopter interviews.
+- Grafana Labs joined the maintainer team.
+
+## The Journey So Far
+
+- More than 600 people have contributed PRs across the OpenFGA org
+- 4,500 stars across OpenFGA repositories
+- Breakout sessions and a kiosk at five KubeCon editions
+- Monthly community meetings running consistently for over 2 years
+- Production adopters include Canonical, Docker, Grafana Labs, Read AI, Agicap, Headspace, Zuplo, and Sourcegraph along with others
+
+## What We Heard from Adopters
+
+ReBAC flexibility for real sharing models, documentation that shortens time to production, responsive maintainers, and an active CNCF Slack community. Adopters also asked for continued progress on idempotent batch writes, DSL ergonomics, public benchmarks, more published adopter stories, and broader maintainer diversity.
+
+## Thank You
+
+A lot of people contributed to OpenFGA's success:
+
+- Massimiliano Gori believed in OpenFGA early and led integration across Canonical.
+- Pauline Jamin spearheaded adoption at Agicap and presented the solution at KubeCon Europe 2024.
+- Andrew Powers led Read AI implementation supporting collaboration and RAG for enterprise search.
+- Joao Guerreiro led the implementation at Grafana and presented their journey at KubeCon Europe 2025.
+- Dan Cech from Grafana contributed the SQLite adapter and serves as a maintainer.
+- Nathan Totten led Zuplo integration, implementing authorization at the edge for the API gateway.
+- Gurleen Sethi led the implementation of organization and team management at Docker, Inc.
+- Siddhant Khare was the first independent contributor to OpenFGA.
+- Maurice Ackel donated the OpenFGA Terraform Provider.
+- Martin Bessozzi and Yann from Agicap.
```

Comment:
Let's add links to all of these people. 

- [Yann D'Isanto](https://www.linkedin.com/in/yann-d-19851110/) from Agicap, who contributed the JetBrains IDE plugin
- [Martin Besozzi](https://www.linkedin.com/in/embesozzi/) has been involved in the OpenFGA community from the beginning, driving integrations with Keycloak.

We should also find everyone who presented in community meetings that are not above and mention them.